packages/slidev/node/options.ts

```diff
@@ -132,7 +137,8 @@ function getDefine(options: Omit<ResolvedSlidevOptions, 'utils'>): Record<string
       __SLIDEV_FEATURE_BROWSER_EXPORTER__: matchMode(options.data.config.browserExporter),
       __SLIDEV_FEATURE_WAKE_LOCK__: matchMode(options.data.config.wakeLock),
       __SLIDEV_HAS_SERVER__: options.mode !== 'build',
+      __SLIDEV_FEATURE_TYPST__: options.data.headmatter.formulaRenderer === 'typst',
```

Comment:
```suggestion
      __SLIDEV_FEATURE_FORMULA_RENDERER__: options.data.headmatter.formulaRenderer,
```

Should it be something like this?